hotsos/defs/scenarios/openvswitch/ovn/ovsdb_reconnect_errors.yaml

```diff
@@ -3,7 +3,7 @@ checks:
     input:
       - var/log/neutron/neutron-server.log
       - var/log/neutron/neutron-ovn-metadata-agent.log
-    expr: '.+ ovsdbapp.backend.ovs_idl.connection ValueError: non-zero flags not allowed in calls to send\(\) on <class ''eventlet.green.ssl.GreenSSLSocket''>'
+    expr: '([\d-]+) [\d:]+\.\d{3} .+ ovsdbapp.backend.ovs_idl.connection ValueError: non-zero flags not allowed in calls to send\(\) on <class ''eventlet.green.ssl.GreenSSLSocket''>'
```

Comment:
you are right, i should be more consistent since that is really the point of this patchset. they are both correct in fact but yeah i will change it to be the same as the others.